### DISCUSSION TITLE
🔗 Add vertical alignment to testimonial carousel widget

### DISCUSSION THREAD

**Discussion by JennyPower** [elementor/elementor#22173] [Feature Request]
### Prerequisites

- [X] I have searched for similar features requests in both open and closed tickets and cannot find a duplicate.
- [X] The feature is still missing in the latest stable version of Elementor ( Elementor Pro )

### What problem is your feature request going to solve? Please describe.

The testimonial carousel requires a feature that enables vertical alignment of the testimonial content. At present, the widget aligns testimonials at the top, leading to an excess of white space between the footer and shorter testimonials. Note that I am not using the image in the testimonial layout

### Describe the solution you'd like

It would be great if an option for vertical alignment were added

**Comment by JennyPower** [comment]
BTW Fix for this:
.swiper-wrapper {
    display: flex;
    align-items: center;
}

**Comment by medieskolenlyngby** [comment]
I found this fix by Maxime Desrosiers that work for me :-)

`.elementor-widget-testimonial-carousel .elementor-testimonial {
align-self: center!important;
}
.elementor-widget-testimonial-carousel .swiper-slide {
display: flex!important;
}
OR
.elementor-widget-testimonial-carousel .elementor-testimonial {
align-items: center!important;
}
.elementor-widget-testimonial-carousel .swiper-slide {
display: flex!important;
}`

**Comment by nicholaszein** [comment]
This discussion was mentioned in:

- #24040
